src/hooks/index.jsx

```diff
@@ -16,3 +17,25 @@ export const useInstitutionIdQueryParam = () => {
 
   return addQueryParam;
 };
+
+export const useToast = () => {
+  const [showToast, setShowToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState('');
+
+  const show = (message) => {
+    setToastMessage(message);
+    setShowToast(true);
+  };
+
+  const hide = () => {
+    setShowToast(false);
+    setToastMessage('');
+  };
+
+  return {
+    showToast,
+    toastMessage,
+    show,
+    hide,
+  };
+};
```

Comment:
```suggestion
/**
 * Custom hook to manage toast notifications
 * @returns {Object} Toast state and methods
 */
const useToast = () => {
  const [toast, setToast] = useState({
    isVisible: false,
    message: ''
  });

  const showToast = (message) => {
    setToast({
      isVisible: true,
      message
    });
  };

  const hideToast = () => {
    setToast({
      isVisible: false,
      message: ''
    });
  };

  return {
    isVisible: toast.isVisible,
    message: toast.message,
    showToast,
    hideToast
  };
};

export default useToast;
```

Reply:
@01001110J comments addressed, thanks for the suggestions.